tests/providers/amazon/aws/triggers/test_base.py

```diff
@@ -63,10 +63,33 @@ def test_region_serialized(self):
         assert "region_name" in args
         assert args["region_name"] == "my_region"
 
-    def test_region_not_serialized_if_omitted(self):
+    @pytest.mark.parametrize("verify", [True, False, pytest.param("/foo/bar.pem", id="path")])
+    def test_verify_serialized(self, verify):
+        self.trigger.verify = verify
         _, args = self.trigger.serialize()
 
-        assert "region_name" not in args
+        assert "verify" in args
+        assert args["verify"] == verify
+
+    @pytest.mark.parametrize(
+        "botocore_config",
+        [
+            pytest.param({"read_timeout": 10, "connect_timeout": 42, "keepalive": True}, id="non-empty-dict"),
```

Comment:
Do we have such example exposed in the docs?

Reply:
Not yet. Let's add it with first migrated Operators. 
Lambda it is a good candidate for first migration:
- Just couple Operators/Sensors
- Have deferrable mode

Reply:
The idea create similar RST file as https://github.com/apache/airflow/blob/main/docs/apache-airflow-providers-amazon/_partials/prerequisite_tasks.rst
Describe generic hook parameters + links to boto3 docs (if required) and include into migrated operators

Reply:
I think it's best to create doc for the base class rather than add the example to all/selected operators

Reply:
Yeah we could create page for base boto3 based operators.
However I think that we also need to have this kind of documentation into the specific operators pages. Because from [auto-API pages](https://airflow.apache.org/docs/apache-airflow-providers-amazon/stable/_api/airflow/providers/amazon/aws/operators/lambda_function/index.html#airflow.providers.amazon.aws.operators.lambda_function.LambdaInvokeFunctionOperator) users would go to the operator specific page.

Reply:
I have a plan create some draft PR during the day, so we could discuss documentation there

Reply:
cool so lets merge?